x-pack/plugins/apm/server/lib/errors/queries.test.ts

```diff
@@ -9,7 +9,7 @@ import { getErrorGroups } from './get_error_groups';
 import {
   SearchParamsMock,
   inspectSearchParams
-} from '../../../../../legacy/plugins/apm/public/utils/testHelpers';
+} from '../../../public/utils/testHelpers';
```

Comment:
not related to this PR, but feels like this file should not be in `public` anyway. there's a bunch of node imports there that will break when it's used in storybook for instance.

Reply:
Pretty sure it does, in fact, break in storybook.